app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,13 @@
+class StaticPagesController < ApplicationController
+  def home
```

Comment:
nếu method ko có content thì em dùng cú pháp này cho ngắn gọn nhé
```suggestion
  def home; end
```